packages/datatrak-web-server/src/app/createApp.ts

```diff
@@ -71,8 +71,13 @@ export function createApp() {
     .get<SingleSurveyResponseRequest>('surveyResponse/:id', handleWith(SingleSurveyResponseRoute))
     .use('signup', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
     // Forward everything else to central server
-    .use('*', forwardRequest(CENTRAL_API_URL, { authHandlerProvider }))
-    .build();
+    .use('*', forwardRequest(CENTRAL_API_URL, { authHandlerProvider }));
+
+  await builder.initialiseApiClient([
+    {entityCode: 'DL', permissionGroupName: 'Admin'},
+  ], true);
```

Comment:
We might want to rely on the public countries list rather than giving public access to all countries